src/applications/gi/components/vet-tec/VetTecFilterBy.jsx

```diff
@@ -30,6 +50,28 @@ class VetTecFilterBy extends React.Component {
       component: this,
     });
 
+  renderProviderFilters = () => {
+    const checkBoxes = Object.keys(this.props.providers)
+      .sort()
+      .map(key => (
+        <div key={key}>
+          <Checkbox
+            checked={this.props.filters.provider.includes(key)}
+            name={key}
+            label={`${key} (${this.props.providers[key]})`}
+            onChange={this.handleProviderFilterChange}
```

Comment:
might be able to simplify this handler by using an arrow function here that has the key in scope instead of relying on pulling it out of the event. 

e.g. 

`onChange={() => this.handleProviderFilterChange(key, this.props.filters.provider.includes(key))}`

should also make it easier to test because you don't have to mock up an event- just the values you care about. 

better to use the state on react than to use what's on the DOM. stuff like `domEvent.target.checked` in react is a code smell.

Reply:
Thanks!